Exercise_A/index.html

```diff
@@ -3,14 +3,28 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
+    <title>Index Vincent</title>
     <meta name="description" content="" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" href="" />
+    <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
     <!-- CODE HERE -->
-
-    <script src="" async defer></script>
+    <header>
+      <h1>This page contains Vincent's Excersizes</h1>
+    </header>
+    <section>
+      <div>Click the giraffe bellow to go to the "About" page</div>
+      <div>
+        <a href="./about.html">
+          <img
+            id="giraffeimg"
+            src="/FTWD_Web_Day_02/assets/Zarafa Raachi9a.png"
+            alt="Giraffe with a human head with a Tunisian flag backdrop"
+          />
+        </a>
+      </div>
+    </section>
+    <script src="app.js" async defer></script>
   </body>
 </html>
```

Comment:
You did a good job adding an anchor tag and linking it to another HTML page. Well done!